### DISCUSSION TITLE
为什么打开maa会报病毒Trojan:Win32/Wacatac.H!ml

### DISCUSSION THREAD

**Discussion by misaka10111** [MaaAssistantArknights/MaaAssistantArknights#6862] [Q&A] [answered]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/97486111/19cc9079-6204-4aec-8930-10c4605b8cfe)
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/97486111/27ac791f-0d3e-4916-a8dd-1f62ab9428e7)
用了几个月了, 直到今天上午都很正常, 晚上突然出现这个问题... (最近也没下载什么乱七八糟的东西啊)
(上图是刚从官网https://maa.plus/ 上下载解压后的文件. 旧的maa也是报一样的病毒, 所以尝试了卸载重装, 无果)

**Comment by ABA2396** [comment, marked as answer]
#6662
